bff-node/src/index.ts

```diff
@@ -0,0 +1,37 @@
+import express, { Express } from 'express';
+import bodyParser from 'body-parser';
+// import swaggerUi from 'swagger-ui-express';
+// import swaggerFile from './swagger_output.json';
```

Comment:
You can consider removing Swagger completely until it needs to be implemented. Then we avoid unused code and future developers do not need to wonder why it's commented out. 😊